examples/line-liff-v1/server.js

```diff
@@ -0,0 +1,46 @@
+const path = require('path');
+
+const bodyParser = require('body-parser');
+const express = require('express');
+const { bottender } = require('bottender');
+const ejs = require('ejs');
+
+const app = bottender({
+  dev: process.env.NODE_ENV !== 'production',
+});
+
+const port = Number(process.env.PORT) || 5000;
+
+const handle = app.getRequestHandler();
+
+app.prepare().then(() => {
+  const server = express();
+
+  server.use(
+    bodyParser.json({
+      verify: (req, _, buf) => {
+        req.rawBody = buf.toString();
+      },
+    })
+  );
+
+  server.get('/liff', (req, res) => {
+    const filename = path.join(`${__dirname}/liff.html`);
+    ejs.renderFile(filename, {}, {}, function(err, str) {
```

Comment:
This HTML render doesn't benefit from the ejs template engine, so we should use `res.sendFile` instead.

Reply:
This should work, if I don't have any misunderstanding:

```js
server.get('/liff', (req, res) => {
  const filename = path.join(`${__dirname}/liff.html`);
  res.sendFile(filename);
});
```

Reply:
If I'm correct, we can just remove `ejs` and uninstall it.

Reply:
cool